src/main/java/com/unb/tracker/web/CourseController.java

```diff
@@ -1,28 +1,37 @@
 package com.unb.tracker.web;
 
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import com.opencsv.CSVWriter;
 import com.unb.tracker.exception.BadRequestException;
 import com.unb.tracker.exception.NotFoundException;
+import com.unb.tracker.model.Absence;
 import com.unb.tracker.model.Course;
 import com.unb.tracker.model.Seat;
 import com.unb.tracker.model.User;
+import com.unb.tracker.repository.AbsenceRepository;
 import com.unb.tracker.repository.CourseRepository;
 import com.unb.tracker.repository.SeatRepository;
 import com.unb.tracker.repository.UserRepository;
 import com.unb.tracker.validator.CourseValidator;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure. EnableAutoConfiguration;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.ModelMap;
 import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import sun.rmi.runtime.Log;
```

Comment:
This line prevented me from starting the application, I had to delete it first

Reply:
I don't even know why this got imported